src/allmydata/test/test_util.py

```diff
@@ -1,5 +1,11 @@
 from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
 
+from future.utils import PY2
+if PY2:
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, pow, round, super, bytes, dict, int, list, object, range, str, max, min  # noqa: F401from builtins import range
```

Comment:
Before merging, discuss this with @itamarst because I think the conclusion he reached for https://github.com/tahoe-lafs/tahoe-lafs/pull/763 is that we should take `int` out of this list for all current and future modules.